### PR TITLE
Code Review for Javascript Project Style Core Elements: Awesome Books

### PR DESCRIPTION
**Pull request by amiraabouhadid**
CSS  was used to style the awesome book's website.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -11,25 +11,41 @@ form {
...
+}
+
+.books-container li:nth-child(even) {
+  background: darkgrey;
+}
+
+.books-container .book-list {
   display: flex;
   flex-direction: column;
   justify-content: center;
```
Comment (mimipeshy) [inline css/main.css]: - [x] Let's keep our code DRY. Justify-content and flex-direction: column has been repeated more than once. To solve this you can group all classes with the same attribute together and give it one attribute eg
```
.books-container .book-list,
form {
     justify-content: center;
}
```

File: css/main.css [hunk 2]
```diff
@@ -11,25 +11,41 @@ form {
...
   list-style: none;
 }
 
-.book-list-element {
+.books-container .book-list-element {
   display: flex;
-  flex-direction: column;
-  justify-content: center;
   align-items: center;
+  flex-direction: row;
```
Comment (mimipeshy) [inline css/main.css]: - [x] Let's keep our code DRY. Align-items: center has also been repeated. Kindly refer to the previous review on how to solve this

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @amiraabouhadid and @gabrielyea  

Good job so far!

#### Highlights

- Good job in following the correct git-flow ✔️ 
- Good job in the UI design, you have kept the styling similar to the given styles ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
------

File: css/main.css (hunk 2 above)
Comment (gabrielyea) [reply css/main.css]: Fixed!

**Review by ZawadiSumba66 — APPROVED** [review]
#### Hi  @amiraabouhadid && @gabrielyea , 

## STATUS: APPROVED 👍 👍 
Good job working on the changes requested by the previous reviewer. Your project is complete and there is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**